include.js

```diff
@@ -1,3 +1,3 @@
 var why = require('./')
 
-process.on('SIGUSR1', function() { why() })
```

Comment:
did you mean to delete this line?

Reply:
Yep, that's what `siginfo` does. Attaches a `SIGUSR1` and `SIGINFO` listener

Reply:
i see the module does this, neat